packages/testsweets/lib/src/models/application_models.dart

```diff
@@ -15,47 +15,37 @@ class WidgetDescription with _$WidgetDescription {
     String? id,
 
     /// The name of the view this widget was captured on
-    required String viewName,
+    String? viewName,
 
     /// The orignal name of the view this widget was captured on before the prettify
-    required String originalViewName,
+    String? originalViewName,
 
     /// The name we want to use when referring to the widget in the scripts
-    required String name,
+    @Default('') String name,
 
     /// The type of the widget that's being added
-    required WidgetType widgetType,
+    WidgetType? widgetType,
```

Comment:
I don't agree with making this not required. There should never be a widget without a type and we want to enfore that through the code and make it clear.

Reply:
I made this nullable inorder to let the user decied its initial value when creating a new widget
But I'll make it required again and put touchable as the initial value